lib/util.js

```diff
@@ -49,7 +49,7 @@ exports.inflate = function(obj) {
     predictor = params.get("Predictor")
   }
 
-  const inflate = require('pako/lib/inflate.js').inflate
+  const inflate = require('pako').inflate
```

Comment:
Thanks for the PR! The idea of using `pako/lib/inflate.js` directly is to make sure the bundler isn't also packaging the deflate part of the library.

I don't get the error you are describing, what bundler are you using?

Reply:
Thanks for the review. Here is my packages info.

```
    "pdf-lib": "^1.16.0",
    "pdf-merger-js": "^3.1.0",
    "pdf-to-printer": "^2.0.5",
    "pdfjs": 
```

Reply:
What are you using to bundle/transpile your app - webpack, vite, parcel, snowpack, something else?

Reply:
I'm using this library in `backend` using `typescript` and only transpile using `tsc`. Is it right?

Reply:
Thanks for the info, I'll try to reproduce the issue with a similar setup 👍

Reply:
Hi @rkusa - I'm getting the same issue. Running tsc, and then using as a package in another app. Same error as above.

Reply:
@rkusa so it was failing on Node 14.5.0 and working on 14.17.3. Thanks.